docs/sources/release-notes/next.md

```diff
@@ -12,3 +12,5 @@ Grafana Labs is excited to announce the release of Loki ?.?. Here's a summary of
 
 ## Features and enhancements
 
+-  **Track effectiveness of hedged requests**: :warning: ADD DESCRIPTION HERE :warning:. [PR #10281](https://github.com/grafana/loki/pull/10281)]
```

Comment:
![Screenshot 2023-08-17 at 15 36 58](https://github.com/grafana/loki/assets/3735252/b0288337-1e7a-4bb9-9ed0-576548f45f92)

This is how it looks now. Either you can add a short description about the feature or remove it if you think just title is good enough.